hermes-api/src/test/java/pl/allegro/tech/hermes/api/SubscriptionTest.java

```diff
@@ -57,6 +57,78 @@ public void shouldDeserializeSubscriptionWithoutBackoff() throws Exception {
         assertThat(subscription.getSerialSubscriptionPolicy().getMessageBackoff()).isEqualTo(100);
     }
 
+    @Test
+    public void shouldDeserializeSubscriptionWithDefaultTracking() throws Exception {
+        // given
+        String json = "{" +
+                "\"name\": \"test\", " +
+                "\"topicName\": \"g1.t1\", " +
```

Comment:
JAVA Y U NO MULTULINE STRING 😢

Reply:
Just wait for Java 12
https://openjdk.java.net/jeps/326